File: challenges/2-idade-em-dias/javascript/LauanAmorim/idadeDias.js
let idadeDias = process.argv;

function Converter(dias) {  
    anos = Math.floor(dias/365);
    meses = Math.floor((dias%365) / 30);
    dias = Math.floor((dias%365) % 30);

    if (meses >= 12)
    {
      anos = 1;
      meses = 0;
      console.log(`${anos} ano(s) \n ${meses} mes(es) \n ${dias} dia(s)`);
    } else
    {
      console.log(`${anos} ano(s) \n ${meses} mes(es) \n ${dias} dia(s)`);
    }
}

Converter(idadeDias[2]);

File: challenges/2-idade-em-dias/python/edurs2602/idadeDias.py
data = int(input('Digite a data em dias: '))

anos = data // 365
meses = (data % 365) // 30
dias = (data % 365) % 30

print(f'{anos} ano(s)\n')
print(f'{meses} mes(s)\n')
print(f'{dias} dia(s)\n')
